typings/README.md

```diff
@@ -1 +1,26 @@
 This folder contains custom type definitions for Typescript.
+
```

Comment:
Can you please update the formatting according to [this doc](https://docs.google.com/document/d/1oYiGMzG42epbK5_8IAPXS_U80r62Xw41NZFBYjsksI4/edit?usp=sharing). You can just copy the content from doc and paste here.

Reply:
Done!